monday/client.py

```diff
@@ -7,23 +7,34 @@
 """
 
 from .__version__ import __version__
-from .resources import ItemResource, UpdateResource, TagResource, BoardResource, UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource
+from .resources import CustomResource, ItemResource, ColumnsResource, UpdateResource, TagResource, BoardResource, \
+    UserResource, GroupResource, ComplexityResource, WorkspaceResource, NotificationResource, MeResource
+
+_DEFAULT_HEADERS = {}
 
 
 class MondayClient:
-    def __init__(self, token):
+    def __init__(self, token, headers=None):
         """
-        :param token: API Token for the new :class:`BaseResource` object.
+        :param token: API token for the new :class:`BaseResource` object.
+        :param headers: (optional) headers for the new :class:`BaseResource` object.
         """
-        self.items = ItemResource(token=token)
-        self.updates = UpdateResource(token=token)
-        self.tags = TagResource(token=token)
-        self.boards = BoardResource(token=token)
-        self.users = UserResource(token=token)
-        self.groups = GroupResource(token=token)
-        self.complexity = ComplexityResource(token=token)
-        self.workspaces = WorkspaceResource(token=token)
-        self.notifications = NotificationResource(token=token)
+
+        if not headers:
+            headers = _DEFAULT_HEADERS.copy()
```

Comment:
Give that this is 2.0.0 and that the goal is to switch to new API, should default headers be `{"API-Version": "2023-10"}` from the get-go?

Reply:
That makes sense to me, done.